domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1107,6 +1107,49 @@ public void optimizedSQLwithNo500(TestContext testContext) {
         }));
   }
 
+   @Test
+  public void deleteByCQLwithNo500(TestContext testContext) {
```

Comment:
All the deleteByCQL tests should go to line 404, right after the deleteById tests, and not between the optimized sql tests.

Reply:
Yes.. Much better! Done now